src/test/resources/bandits/rac-experiments-bandits-beta.json

```diff
@@ -38,6 +38,44 @@
         }
       }
     },
+    "uninitialized-bandit-experiment": {
+      "subjectShards": 10000,
+      "overrides": {},
+      "typedOverrides": {},
+      "enabled": true,
+      "rules": [
+        {
+          "allocationKey": "bandit",
+          "conditions": []
+        }
+      ],
+      "allocations": {
+        "bandit": {
+          "percentExposure": 0.4533,
+          "variations": [
+            {
+              "name": "control",
+              "value": "control",
+              "typedValue": "control",
+              "shardRange": {
+                "start": 0,
+                "end": 2000
+              }
+            },
+            {
+              "name": "bandit",
+              "value": "this-bandit-does-not-exist",
+              "typedValue": "this-bandit-does-not-exist",
+              "shardRange": {
+                "start": 2000,
+                "end": 10000
+              },
```

Comment:
I'm a bit confused what is being tested using this. IIUC, it's "a flag that will contain a bandit has been created, but the bandit itself hasn't been created". That feels like a weird edge case, especially with our creation flow.

A more important case is when the SDK attempts to get a bandit action for which no bandit exists, the flag that contains it doesn't exist, etc. It's just not in the RAC at all. Are we testing that case? (Please let me know if this question doesn't make sense – I might be misunderstanding the whole architecture.)

Reply:
We had that test already--I've been calling it an "uninitialized bandit". ([source](https://github.com/Eppo-exp/java-server-sdk-bandit-beta/blob/aaron/ff-1599/test-for-cold-start/src/test/java/com/eppo/sdk/EppoClientTest.java#L328))

This newly added "cold start" test is making sure we handle the more likely case that a user creates a bandit in our UI so the bandit exists, but it hasn't been trained yet.